docs/send-data/use-json-configure-sources/local-configuration-file-management/index.md

```diff
@@ -80,24 +78,24 @@ Source in a `sources` JSON array.
 The maximum number of Sources allowed on a Collector is 1,000.
 :::
 
-To set up new Collectors and Sources see [Local Configuration File Management for New Collectors and Sources](new-collectors-and-sources.md). 
+To set up new Collectors and Sources, see [Local Configuration File Management for New Collectors and Sources](new-collectors-and-sources.md). 
 
-To convert existing Sources from Cloud Management to Local Configuration File Management see [Local Configuration File Management for Existing Collectors and Sources](existing-collectors-and-sources.md).
+To convert existing Sources from Cloud Management to Local Configuration File Management, see [Local Configuration File Management for Existing Collectors and Sources](existing-collectors-and-sources.md).
 
 ### Configure the location of JSON file or folder
 
 When using local file configuration management you specify the location of the JSON file or the folder that contains multiple JSON files in the Collector’s `config/user.properties` file. You need to use the `syncSources` parameter to point to your configuration file or folder. See `user.properties` for details on all of the available parameters.
 
-* **syncSources**. Use to define the Sources to configure upon Collector registration. The Source definitions will be continuously monitored and synchronized with the Collector's configuration. 
-
-  * On \*nix, to point to a JSON file that defines Sources for a Collector:
-
-    `syncSources=/path/to/sources.json`
-
-  * On \*nix, to point to a folder that contains JSON files that define Sources for a Collector:
-
-    `syncSources=/path/to/sources-folder`
-
-  * On Windows (note the escaped backslashes), to point to a folder that contains JSON files that define Sources for a Collector:
-
-    `syncSources=C:\\path\\to\\sources-folder\\`
+**syncSources**. Use to define the Sources to configure upon Collector registration. The Source definitions will be continuously monitored and synchronized with the Collector's configuration. 
+* On \*nix, to point to a JSON file that defines Sources for a Collector:
+   ```
+   syncSources=/path/to/sources.json
+   ```
+* On \*nix, to point to a folder that contains JSON files that define Sources for a Collector:
```

Comment:
Again, is "\*nix" supposed to be "Linux"?

Reply:
https://github.com/SumoLogic/sumologic-documentation/pull/3588#discussion_r1478888165